.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ orbs:
 parameters:
   alerts-slack-channel:
     type: string
-    default: dps_alerts_security
+    default: syscon-alerts
```

Comment:
@andymarke do you think you'll move other services to this channel e.g. other api syscon support?